### PR TITLE
Changed printf

### PR DESCRIPTION
**Pull request by cmthatch**
# Title

## Sub-title

Take that!

Like this:

~~~ {.c}
   // Here's my comment
   printf("Yeah, right");
~~~

### CODE REVIEW

File: hello.c [hunk 1]
```diff
@@ -4,5 +4,5 @@
 
 void printHelloGoodbye( void )
 {
-   printf("HELLO, WORLD.");
+   printf("HELLO, Solar System.");
```
Comment (joebobfisher) [inline hello.c]: No, this is all wrong. Did you even read the spec???

Comment (cmthatch) [inline hello.c]: What spec?  You never sent me one...

Comment (joebobfisher) [inline hello.c]: It doesn't matter, but it needs to be in all caps.

**Comment by joebobfisher** [comment]
Make this all caps, and we're good.

**Pull request closed without merging**